Rename FrameView::m_size to m_lastUsedSizeForLayout

smfr opened the pull request:
#### 56bc3a69c714260a2c0de11763429618429e8db4
<pre>
Rename FrameView::m_size to m_lastUsedSizeForLayout
<a href="https://bugs.webkit.org/show_bug.cgi?id=179579">https://bugs.webkit.org/show_bug.cgi?id=179579</a>

Reviewed by Alan Bujtas.

Rename FrameView::m_size to m_lastUsedSizeForLayout.

* Source/WebCore/page/FrameView.cpp:
(WebCore::FrameView::init):
(WebCore::FrameView::willDoLayout):
(WebCore::FrameView::updateScriptedAnimationsAndTimersThrottlingState):
* Source/WebCore/page/FrameView.h:

Canonical link: <a href="https://commits.webkit.org/252894@main">https://commits.webkit.org/252894@main</a>
</pre>

alanbaradlay reviewed (approved):
discussed naming change offline

smfr committed:
Rename FrameView::m_size to m_lastUsedSizeForLayout
https://bugs.webkit.org/show_bug.cgi?id=179579

Reviewed by Alan Bujtas.

Rename FrameView::m_size to m_lastUsedSizeForLayout.

* Source/WebCore/page/FrameView.cpp:
(WebCore::FrameView::init):
(WebCore::FrameView::willDoLayout):
(WebCore::FrameView::updateScriptedAnimationsAndTimersThrottlingState):
* Source/WebCore/page/FrameView.h:

Canonical link: https://commits.webkit.org/252894@main
